phd2.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Mattia Verga <mattia.verga@tiscali.it> -->
+<component type="desktop-application">
+    <id>phd2.desktop</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
+    <name>Comical</name>
+    <summary>Telescope guiding software</summary>
+    
+    <description>
+        <p>
+        PHD2 is telescope guiding software that simplifies the process
+        of tracking a guide star, letting you concentrate on other aspects of
+        deep-sky imaging or spectroscopy.
+        </p>
+        <ul>
+            <li>Easy-to-use, “push here dummy” guiding for beginners</li>
+            <li>Sophisticated guiding and analysis tools for experienced users</li>
+            <li>Extensive support for commonly-used equipment</li>
+            <li>Easy upgrade/co-installation with PHD1, including many new features</li>
+            <li>Extensive online help and responsive product support</li>
+        </ul>
+    </description>
+    
+    <launchable type="desktop-id">phd2.desktop</launchable>
+    
+    <screenshots>
+        <screenshot type="default">
+            <caption>The main window</caption>
+            <image>http://openphdguiding.org/wp-content/uploads/2015/06/capture_21.jpg</image>
+        </screenshot>
+        <screenshot>
+            <caption>The log viewer</caption>
+            <image>http://openphdguiding.org/wp-content/uploads/2015/05/phdlogviewer_sm.png</image>
```

Comment:
Ah, I thought it was included in the main PHD2 program... I will remove this screenshot.